.github/ISSUE_TEMPLATE/2-feature-request.yml

```diff
@@ -0,0 +1,41 @@
+name: Feature Request
+description: Create a proposal to request a feature
+title: "[Feature]: "
+labels: ["T:feature-request"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+       ✰  Thanks for opening an issue! ✰  
+  - type: textarea
+    id: summary
+    attributes:
+      label: Summary
+      description: |
+        What are the user needs?
+        How could this solution fix the user facing problem? 
+      placeholder: Short, concise description of the proposed feature/changes to the repository
+    validations:
+      required: true
+  - type: textarea
+    id: problem
+    attributes:
+      label: Problem Definition
+      description: |
+        If applicable please answer the below questions
+        Why do we need this feature? 
```

Comment:
I would suggested to not use the word `problem` in the context of a feature request